sdk/nodejs/api/client.gen.ts

```diff
@@ -681,6 +681,10 @@ Used for multi-platform image.
     return response
   }
 
+  with(arg: (param: Container) => Container) {
```

Comment:
The arrow function is needed since we work on the `this,` but it creates inconsistency on the SDK.
Should we update every function to arrow style, or is this inconsistency allowable?

Reply:
No you're right. It would be better to stay consistant. We should switch to arrow style on every method.

Reply:
Okay, I keep track of this in #4294

Reply:
Oh, wait, I might have misunderstood the code.

The arrow function is used as a type, not as a value. The `with` function is still a classic function!

We're still consistent. My bad.

Reply:
Yes the `with` function is still a classic function.